src/Explorer/Tabs/Tabs.tsx

```diff
@@ -116,6 +117,16 @@ function TabNav({ tab, active, tabKind }: { tab?: Tab; active: boolean; tabKind?
             {isTabExecuting(tab, tabKind) && (
               <img className="loadingIcon" title="Loading" src={loadingIcon} alt="Loading" />
             )}
+            {isQueryErrorThrown(tab, tabKind) ? (
```

Comment:
Any reason not to make this isQueryErrorThrown(...) && ( <img ... /> ) so it's consistent with the previous statement ?

Reply:
No reason there, thanks for pointing out! :) Will fix that now.